columnflow/production/cms/muon.py

```diff
@@ -93,7 +92,7 @@ def muon_weights(
         weight = ak.prod(sf, axis=1, mask_identity=False)
 
         # store it
-        events = set_ak_column(events, f"muon_weight{postfix}", weight, value_type=np.float32)
+        events = set_ak_column(events, f"{self.weight_name()}{postfix}", weight, value_type=np.float32)
```

Comment:
The `self.weight_name()` function is already evaluated in the init function. We might want to cache the name to avoid multiple function calls (though this of course is completely irrelevant if the function is not super heavy, which it most likely won't be I guess)

Reply:
Not necessary anymore since `sel.weight_name` is just a simple string now